cif/shoppinglists/src/mappers/ShoppingListEntryMapper.ts

```diff
@@ -0,0 +1,92 @@
+/*
```

Comment:
Review if it's possible to use CartEntryMapper

Reply:
Although they are similar, there are a few differences between the CartEntry and ShoppingListEntry data structures (name of same fields, the location where the price components are used, etc) so I think it's best to keep them in separate files